README.md

```diff
@@ -182,64 +191,134 @@ require("jupynium").setup({
+- To sync a Jupynium file to an existing notebook, manually open the file in the browser, and `:JupyniumStartSync 2` to sync to the 2nd tab (count from 1).
 
-- This will not open a file if it exists. Instead filename will be ignored.
+At this point, any changes you make within the Neovim Jupynium file will be reflected live in the browser. Make sure you do not make changes inside the browser itself, as the sync is only one-way (from Neovim to browser).
 
-2. Manually open the file from the browser, and `:JupyniumStartSync 2` to sync using the 2nd tab (count from 1).
+If you want to save a copy of the `.ipynb` file, run `:JupyniumSaveIpynb`. There is also a configuration option to enable automatic saving/downloading.
 
-### Use multiple buffers
+#### Sync multiple Jupynium files
 
-Run `:JupyniumStartSync` again with a new buffer you want to sync.
+You can sync multiple files at the same time. Simple run `:JupyniumStartSync` again with the new file you want to sync.
 
-### Use multiple neovim
+#### Use multiple Neovim
 
-You can run `:JupyniumStartSync` on a new neovim instance.  
+You can run `:JupyniumStartSync` on a new Neovim instance.  
 If you have `auto_attach_to_server = false` during setup, you need to run `:JupyniumAttachToServer` and `:JupyniumStartSync`.
 
-## 📝 .ju.py (or .ju.\*) file format
+## 👨‍💻️ Command-Line Usage (attach to remote Neovim)
+
+**You don't need to install the vim plugin to use Jupynium.**  
+**The plugin is responsible of adding `:JupyniumStartAndAttachToServer` etc. that just calls the command line program, plus it has textobjects and shortsighted support.**
+
+Install Jupynium if you haven't already:
+
+```bash
+pip3 install jupynium
+```
+
+Open a python/markdown file with nvim and see `:echo v:servername`.  
+Run Jupynium like:
+
+```bash
+jupynium --nvim_listen_addr /tmp/your_socket_path
+```
+
+Or, you can run Neovim like
+
+```bash
+nvim --listen localhost:18898 notebook.ju.py
+```
+
+Then start Jupynium as well as attaching the neovim to it.
+
+```bash
+jupynium --nvim_listen_addr localhost:18898
+```
+
+Note that you can attach to remote neovim by changing `localhost` to `servername.com` or using SSH port forwarding.
+
+This will open Firefox with Selenium, defaulting to `http://localhost:8888`.
+
+Additionally,
+
+- You can run `jupynium` command multiple times to attach more than one Neovim instance.
+- `jupynium --notebook_URL localhost:18888` to view different notebook.
+- You can just run `jupynium` without arguments to just leave the server / browser running and wait for nvim to attach.
+
+## 📝 Jupynium file format (.ju.py or .ju.\*)
 
-The file format is designed to be LSP friendly even with markdown code injected in it. The markdown cells will be part of python string `"""%%` ... `%%"""`.
+The file format is designed to be LSP friendly even with markdown code injected into it. The markdown cells will be part of a Python docstring: `"""%%` ... `%%"""`.
```

Comment:
Docstring means it's a string at the first line of a module definition. You can just say multi-line string or just leave it as string